nepal_compliance/nepali_date_utils/nepali_date.py

```
@@ -107,18 +100,13 @@ def _ensure_loaded():
         load_calendar()
 
 
-def days_in_bs_month(year: int, month: int) -> int:
-    """Number of days in BS year-month (1-indexed month)."""
+def _validate_bs(year: int, month: int, day: int):
     _ensure_loaded()
     if year not in _bs_months:
         _throw(f"Year {year} outside supported BS range {min(_bs_months)}–{max(_bs_months)}")
```

Ruff (0.16.2)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)